src/sidebar/components/publish-annotation-btn.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
       return this.isShared ? this.group.name : this.privateLabel;
     };
 
-    this.groupType = function () {
-      return this.group.public ? 'public' : 'group';
+    this.groupCategory = function () {
```

Comment:
I renamed this method to try to avoid confusion, as it's not really about type so much as a sort of "category" of groups. `category` might not be quite right, either, but I hope it will avoid total confound-ment of future devs.